packages/ubus-lime-utils/Makefile

```diff
@@ -23,7 +23,6 @@ endef
 define Package/$(PKG_NAME)/install
 	$(INSTALL_DIR) $(1)/
 	$(CP) ./files/* $(1)/
-	@chmod a+x $(1)/usr/libexec/rpcd/lime-utils
```

Comment:
I was trying out this branch and founded out that we missed to setup execution permissions to lime-utils-admin.
Let's add it in this PR?

Reply:
I opened this PR in case we prefer to address this issue in its own PR: https://github.com/libremesh/lime-packages/pull/704

Reply:
Thanks! I cherry picked your commit